### ISSUE TITLE
-mcrt=clib2-ts (thread safe variant) required

### ISSUE THREAD

**Issue by jens-maus** [cahirwpz/amigaos-cross-toolchain#68]
For certain projects we do require the thread safety support of clib2 which however means that clib2 would have to be compiled with different compiler flags (`-D__THREAD_SAFE`).

For convenience and to not impact processing performance you could simply compile a "clib2-ts" variant together with the normal "clib2" version you supply anyway. To select the right clib2 version a developer could then easily choose the thread safe variant of clib2 by specifying `-mcrt=clib2-ts` which in fact we would do.

**Comment by jens-maus** [comment]
Any chance to have that implemented any time soon?

**Comment by cahirwpz** [comment]
In fact this task is next on my private list. Do I understand correctly that if `-mcrt=clib2-ts` is implemented then issue #67 will get invalidated?

**Comment by jens-maus** [comment]
No, #67 is a different issue with the `-L` link options not taking preference anymore. This has nothing to do with the possibility to have a `-mcrt=clib2-ts` option to have a thread-safe clib2 variant.

**Comment by emartisoft** [comment]
I am sorry, I could not follow this issue.
While I am using `popen()` & `pclose()` function in `<stdio.h>`, not compiles `-noixemul` options. But It compiles via using` -mcrt=clib2` option in makefile.
Is it possible that compile with` -noixemul`?

**Comment by jens-maus** [comment]
@emartisoft Sorry, but what does your observation have to do with this ticket? This ticket is about implementing support for the thread-safe variant of the clib2 runtime library because currently this toolchain only provided clib2 without thread-safety support which for some projects (e.g. AmiSSL) is utterly required.
